src/base/bittorrent/torrenthandleimpl.cpp

```diff
@@ -125,8 +125,8 @@ TorrentHandleImpl::TorrentHandleImpl(Session *session, const lt::torrent_handle
     , m_seedingTimeLimit(params.seedingTimeLimit)
     , m_hasSeedStatus(params.hasSeedStatus)
     , m_hasRootFolder(params.hasRootFolder)
-    , m_useAutoTMM(params.savePath.isEmpty())
     , m_hasFirstLastPiecePriority(params.firstLastPiecePriority)
+    , m_useAutoTMM(params.savePath.isEmpty())
```

Comment:
Unrelated change

Reply:
Or is it? If this doesn't have any implications for CI or build success/failure, it should be in another "code cleanup" PR IMO.

Reply:
The defect is found along when fixing the CI, it is usually acceptable to have such small/harmless fixes along the way.

Reply:
@FranciscoPombal
Yes, we usually allow such trivial changes to be in an unrelated PR. The main thing is that it should be as a separate commit. There is nothing to discuss about it to make a separate PR for it purposely.